rules/integrations/o365/initial_access_o365_user_reported_phish_malware.toml

```diff
@@ -0,0 +1,60 @@
+[metadata]
+creation_date = "2022/01/12"
+integration = "o365"
+maturity = "production"
+updated_date = "2022/01/12"
+
+[rule]
+author = ["Elastic"]
+description = """
+Detects the occurrence of emails reported as Phishing or Malware by Users. Security Awareness training is essential to
+stay ahead of scammers and threat actors, as security products can be bypassed, and the user can still receive a
+malicious message. Educating users to report suspicious messages can help identify gaps in security controls and prevent
+malware infections and Business Email Compromise attacks.
+"""
+false_positives = ["Legitimate files reported by the users"]
+from = "now-30m"
+index = ["filebeat-*", "logs-o365*"]
+language = "kuery"
+license = "Elastic License v2"
+name = "Email Reported by User as Malware or Phish"
```

Comment:
Should add O365 somewhere in here to be consistent with the other integration rules. (Actually all of the rules for this integration need the names normalized)

Reply:
Added it to the start